R/AddCardModule.R

```diff
@@ -4,9 +4,10 @@
 #'
 #' For more details see the vignette: `vignette("simpleReporter", "teal.reporter")`.
 #' @param id `character(1)` this `shiny` module's id.
+#' @param label `character(1)` label before the icon.
 #' @return `shiny::tagList`
 #' @export
-add_card_button_ui <- function(id) {
+add_card_button_ui <- function(id, label = "") {
```

Comment:
lets remove the label here and give a static label when hovering over the button.